sql/cat_tools.sql

```diff
@@ -346,6 +346,8 @@ SELECT __cat_tools.create_function(
   , OUT row_statement text
   , OUT when_clause text
   , OUT function_arguments text
+  , OUT function_name text
```

Comment:
Could be! 
I followed the name used in the [PostgreSQL doc.](https://www.postgresql.org/docs/9.5/static/sql-createtrigger.html) But its up to you. It was just a suggestion.